brainstormforce/ultimate-addons-for-gutenberg
# Fix: Review Schema - Trash icon CSS conflicting with Presto Player.

### Description

- Updated CSS priority of Review trash and plus icon.

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Install plugin - https://wordpress.org/plugins/presto-player/ 
- Drag and drop any presto block
- Add custom Preset
- Add a Review block and hover on added preset you will see a trash icon not visible.
- Switch to `conflict-with-presto-player-trash-icon` issue will be resolved.

### Checklist:
